code/data/create_electricity_json.py

```diff
@@ -31,6 +32,24 @@ def list_to_string(l):
 elecDf = df.loc[(df["Indicator Code"] == "1.2_ACCESS.ELECTRICITY.RURAL") | \
           (df["Indicator Code"] == "1.1_ACCESS.ELECTRICITY.TOT")]
 
+#additional countries to add
+stats = [NaN for i in range(1990, 2017)]
+to_add = [["Scotland", "SCT", "", ""]+stats,
```

Comment:
You should try to find other lists to compare our list of countries too. We missed these which are not very uncommon countries so this might indicate we are missing lesser-known countries.

Reply:
It's true that would be a better way to do it. We missed them because I figured we didn't need to add anything to the database from countries that are just going to be NaNs, thinking that the front end would simply display something like "No data available for this country" if the database didn't return them anything. We are using CIA data, so actually these countries aren't being neglected, but rather they're seen as parts of other countries -- which actually causes incoherencies. I hadn't thought about that before, but I will mention it to the team on slack. 

As far as the code that is here goes, it is only a few countries we needed to add, and due to the end of the semester I am a bit preoccupied with other things at the moment. I'll add a comment for the future group that they should probably write additional countries as csv data.